### ISSUE TITLE
Error sharing as iframe GeoStory or Dashboar

### ISSUE THREAD

**Issue by ElenaGallo** [geosolutions-it/MapStore2#8478]
## Description

It is not possible to share as iframe a geostroy or dashboard

## How to reproduce

-  Login as admin
- Open a geostory/dashboard
- Click on Share
- Click on Embed
- Copy the iframe 
- Past on a new html page (en example here: 
[dash.zip](https://github.com/geosolutions-it/MapStore2/files/9321015/dash.zip))

*Expected Result*

It is possible to see the embed dashboard or geostory

- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>

(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

## Other useful information

It seems a regression introduced after #8457. The tool works well in QA.

**Comment by tdipisa** [comment]
@alexander-fedorenko I see this issue is connected to #8503 now testable in DEV since #8495 is now merged. It seems anyway a server side issue where embeddign through iframe is not allowed at the moment for some reason and not something connected to the modular plugin work. Therefore, I'm wondering what you have fixed for this.

**Comment by alexander-fedorenko** [comment]
@tdipisa Initial issue was that dashboard and geostory was not rendered in iframe at all, there was a white blank page instead of the content. This issue is seems to be a different one and it also is not getting reproduced locally with default proxy configuration.

**Comment by tdipisa** [comment]
>This issue is seems to be a different one and it also is not getting reproduced locally with default proxy configuration.

This is quite strange I think.

In QA the problem is not present anyway, just checked.

**Comment by tdipisa** [comment]
@lpasquali 

> @alexander-fedorenko I see this issue is connected to #8503 now testable in DEV since #8495 is now merged. It seems anyway a server side issue where embeddigng through iframe is not allowed at the moment for some reason and not something connected to the modular plugin work. Therefore, I'm wondering what you have fixed for this.
> 
> ![image](https://user-images.githubusercontent.com/1280027/187418031-0c8edd4a-72f2-4c60-8c41-753b4bf8233f.png)

@lpasquali can you please check DEV config please?
Last time @randomorder fixed the embedding with iframe via HTTP POST for DEV but via HTTP GET it was working. Maybe he broke something for GET?

In QA we have:

POST not working
GET working

It would be good to fix also QA possibly

**Comment by lpasquali** [comment]
@tdipisa could you try now?

**Comment by tdipisa** [comment]
@lpasquali rthe bheavior is the same as before.

**Comment by ElenaGallo** [comment]
@tdipisa 
> @alexander-fedorenko I see this issue is connected to #8503 now testable in DEV since #8495 is now merged. It seems anyway a server side issue where embeddigng through iframe is not allowed at the moment for some reason and not something connected to the modular plugin work. Therefore, I'm wondering what you have fixed for this.
> 
> ![image](https://user-images.githubusercontent.com/1280027/187418031-0c8edd4a-72f2-4c60-8c41-753b4bf8233f.png)

this error is still present on DEV

**Comment by tdipisa** [comment]
@ElenaGallo there is currently an issue in our webserver (@lpasquali knows about this) that is preventing the embedding via Iframe from localhost. If you test this functionality from an online domain like our demo site, it is working well. I will raise a separate issue to ask devops to make it working again from localhost for development purposes.
